platform/ui/src/views/StudyList/components/StudyListTable.js

```diff
@@ -50,54 +54,65 @@ const TableRow = () => {
                   onClick={toggleRow}
                 >
                   <td className={classnames(...tdClasses)}>
-                    <ChevronIcon />
+                    <Icon name={ChevronIconName} />
+                  </td>
+                  <td className={classnames(...tdClasses)}>{PatientName}</td>
+                  <td className={classnames(...tdClasses)}>{PatientId}</td>
+                  <td className={classnames(...tdClasses)}>
+                    {format(StudyDate, 'MMM-DD-YYYY')}
```

Comment:
I think we'll force the parent to handle this formatting. You can ditch `date-fns` here.